src/main/java/com/team2/flea_market/dto/comment/CommentDto.java

```diff
@@ -0,0 +1,19 @@
+package com.team2.flea_market.dto.comment;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public record CommentDto(
+        @Schema(description = "id автора объявления")
+        Integer author,
+        @Schema(description = "ссылка на аватар автора комментария")
+        String authorImage,
+        @Schema(description = "Имя создателя комментария")
+        String authorFirstName,
+        @Schema(description = "дата и время создания комментария в миллисекундах с 00:00:00 01.01.1970")
+        Integer createdAt,
```

Comment:
```suggestion
        long createdAt,
```
int64 - это long, int32 - это int